### PR TITLE
Add Zoom

### PR DESCRIPTION
**Pull request by jimmyvanau**
Add Zoom as a site + link to documentation (https://support.zoom.us/hc/en-us/articles/360038247071-Setting-up-and-using-two-factor-authentication)

### CODE REVIEW

File: _data/communication.yml [hunk 1]
```diff
@@ -355,3 +355,10 @@ websites:
     twitter: Yammer
     facebook: Yammer
     img: yammer.png
+
+  - name: Zoom
+    url: https://www.zoom.us
+    img: zoom.png
+    tfa:
+      - totp
+    doc: https://support.zoom.us/hc/en-us/articles/360038247071-Setting-up-and-using-two-factor-authentication
```
Comment (jamcat22) [inline _data/communication.yml]:
```suggestion
    doc: https://support.zoom.us/hc/en-us/articles/360038247071
    exception: "2FA is only available on paid plans and doesn't protect the desktop or mobile apps."
```

**Review by jamcat22 — CHANGES_REQUESTED** [review]

**Review by jamcat22 — APPROVED** [review]
Thanks @jimmyvanau!

**Comment by Carlgo11** [comment]
Is 2FA completely ignored when signing in through a desktop or phone app?

**Comment by jimmyvanau** [comment]
Looks that way, 2FA only for web settings

**Comment by Carlgo11** [comment]
@jamcat22 is that even 2FA then?

**Comment by prkls** [comment]
> @jamcat22 is that even 2FA then?

Arguably, yes. The 2FA is to the web console where changes to the user's defaults are done. Also, if you're an admin or owner of the Zoom account, any changes to the environment, adding and deleting users, etc. can be 2FA protected.

**Comment by jamcat22** [comment]
I'd say adding it with the exception noted is good. It at least adds some protection to admin settings and recordings. The app now integrating chat features makes it a larger target than when it just was used for joining meetings, but that doesn't mean 2FA should be disabled altogether.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**